SimPEG/EM/Analytics/NSEM.py

```diff
@@ -0,0 +1,128 @@
+import numpy as _np
```

Comment:
At some level, I think that you should be accessing this from the `geoana. <tab>`? Does that work as expected (perhaps commenting out the `import *`'s for testing...)?

When you access the module directly, I think this is opening that module and loading everything. It also sounds like the IPyKernel does not respect the `__all__`, although you can configure it to that if you want:

https://github.com/ipython/ipykernel/issues/129

![image](https://user-images.githubusercontent.com/913249/26959910-63b18f86-4c90-11e7-82f7-e1d29b7e3b92.png)